src/test/cls_lock/test_cls_lock.cc

```diff
@@ -502,10 +502,11 @@ TEST(ClsLock, TestExclusiveEphemeralBasic) {
   ASSERT_EQ(0, l1.lock_exclusive_ephemeral(&ioctx, oid1));
   ASSERT_EQ(0, ioctx.stat(oid1, &size, &mod_time));
   sleep(2);
+  bool r1_expiered = lock_expired(&ioctx, oid1, lock_name1);
```

Comment:
This is a small spelling fix:
```suggestion
  bool r1_expired = lock_expired(&ioctx, oid1, lock_name1);
```

Reply:
Done!